app/client/views/homes/assignManger/assignManager.html

```diff
@@ -0,0 +1,34 @@
+<template name="assignManager">
+  <div class="modal fade">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        {{# if Template.subscriptionsReady }}
+        <div class="modal-header">
+          {{> flashMessages}}
+          <h2 class="modal-title">
+            <i class="fa fa-home"></i>
+            {{_ "assignManager-header-text" }}
+          </h2>
+        </div>
+        <div class="modal-body">
+          {{#if currentAssignUser}}
+          <div>{{_ "assignManager-currentManager-text"}} {{ currentAssignUser }}</div>
+          {{else}}
+          <div>{{_ "assignManager-noCurrentManager"}} !</div>
```

Comment:
Lets not yell at our users.

Reply:
You mean we remove the div showing current managers?

Reply:
Just not to use exclamation mark in text user-facing texts, as it is percieved as shouting at people. E.g.

- warning!
- do not enter!

Those both use exclamation mark to exclaim that there is danger, to impart urgency. Our user texts don't have that level of urgency, so can just state the facts and use a period.

Reply:
Oh I get it now.